policyengine_us/parameters/gov/states/mi/tax/income/deductions/retirement_benefits/tier_three/ss_exempt/retired/birth_year.yaml

```diff
@@ -0,0 +1,27 @@
+description: Michigan allows for an retired tier three retirement and pension benefits for filers born on or after this year. 
+values:
+  2021-01-01: 1955
+  2022-01-01: 1956
+  2023-01-01: 1957
+metadata:
+  unit: year
+  period: year  
+  label: Michigan retired tier three retirement and pension benefits birth year  
+  reference:
+    - title: Michigan Legal Code Section 206.30 - (9)(c)
+      href: http://legislature.mi.gov/doc.aspx?mcl-206-30
```

Comment:
this subsection does not show these years - does the law represent it differently?

Reply:
Not really, I added these birth years only because 2023 tax rule instruction updates a new part and changes the birth years in old tier3:
https://github.com/PolicyEngine/policyengine-us/pull/4258#issuecomment-2014189778
But I couldn't find the rules for this tier3 retired part in Michigan legislature.

Reply:
@Mshuning lets remove the birth year parameter files for now (in the tier 3) as they are equivalent to the age requirement that we already have (and we would have to update each year). Not sure why the forms would add this layer of complexity